packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -259,6 +272,13 @@ class FlutterErrorDetails extends Diagnosticable {
   /// This won't be called if [stack] is null.
   final IterableFilter<String> stackFilter;
 
+  /// A creator that is responsible for this error.
+  ///
+  /// This property must be in a type [FlutterDebugCreatorDescriber] can understand.
+  ///
+  /// If not null, this property will be used for tracing back user created widget.
```

Comment:
this level of the framework doesn't know about widgets, so mentions of widgets in the docs should be only in terms of "for example, in the widgets layer...".

Reply:
(or similar)